copter/source/docs/flight-modes.rst

```diff
@@ -45,6 +45,7 @@ companion computer.
    <tr><td>SysID</td><td>-</td><td>+</td><td></td><td>Special diagnostic/modeling mode</td></tr>
    <tr><td>Throw</td><td>A</td><td>A</td><td>Y</td><td>Holds position after a throwing takeoff</td></tr>
    <tr><td>ZigZag</td><td>A</td><td>A</td><td>Y</td><td>Useful for crop spraying</td></tr>
+   <tr><td>Autonomous Autorotation</td><td>A</td><td>A</td><td>Y</td><td>Used for emergencies in traditional helicopters. Helicopter only.  Currently SITL only.</td></tr>
```

Comment:
placement should be alphabetical in the lists...

Reply:
Moved the autorotation mode to be in its alphabetical position in all lists on this page.